root/scripts/talker/gambler.txt

```diff
@@ -3272,6 +3274,13 @@ Rule EllisStoryInterruptC5M5Gambler
 	Response EllisStoryInterruptC5M5Gambler
 }
 
+Rule EllisStoryInterruptC13M4Gambler
```

Comment:
There's no info about a helicopter that is known to the survivors until after they leave the saferoom. Even then, they would have to interact with the radio to hear said helicopter.

Reply:
There is no mention of a helicopter in Ellis' story, or in Nick' response. The context of Ellis telling a stroy about the army bombing his budddy keith, is that the survivors were themselves bombed by the military in the junkyard of the previous map. The context for Nick saying "If I listen to this, will this be the last one?" is a reference to Cold Stream being the last official appearance of the L4D2 survivors. (It's also the exact same reply Nick makes to the same Ellis story on The Parish's finale, which itself was the last canonical appearance of the L4D2 survivors)